packages/connectivity/example/lib/main.dart

```diff
@@ -46,7 +46,11 @@ class _MyHomePageState extends State<MyHomePage> {
     initConnectivity();
     _connectivitySubscription =
         _connectivity.onConnectivityChanged.listen((ConnectivityResult result) {
-      setState(() => _connectionStatus = result.toString());
+      if (result == ConnectivityResult.wifi) {
+        _getWifiDetails(result.toString());
+      } else {
+        setState(() => _connectionStatus = result.toString());
```

Comment:
As you removed the `'Connection Status: $_connectionStatus\n'` from the `build`, when result != wifi, it won't have a `'Connection Status: '` text anymore.

Actually, I think the best would be placing it back in the `build` method and removing the 'Connection Status:' text from the `getWifiDetails`.